### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/2024/1131229%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#13] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/2024/1131229%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

圖片來源 吳佳玲(2012) 【聖經小百科】金燈臺 https://cdn-news.org/News.aspx?EntityID=News&PK=0000000000652e59f0b526028ed61b418deddf2488f4834c

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/2024/1131229%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
約翰福音8:12，「我是世界的光。跟從我的，就絕不在黑暗裡走，必要得著生命的光。」

詩篇119:105，「你的話是我腳前的燈，是我路上的光。」
